bin/publish.js

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/env node
+
+'use strict';
+/* eslint-disable node/no-unsupported-features/es-syntax, no-console, no-process-exit, node/no-extraneous-require, node/no-unpublished-require */
+
+/*
+Usage
+
+publish release|beta|canary
+
+Flags
+
+--distTag=latest|lts|beta|canary|release-<major>-<minor>
```

Comment:
Whats latest?

Reply:
`latest` is the latest stable release. It is what get's installed by `yarn` or `npm` if you do not otherwise specify a version or tag. It is also the tag that gets published to npm if we do not specify one.